tools/serverpod_cli/test/analyzer/models/stateful_analyzer/model_validation/field/field_validation_test.dart

```diff
@@ -367,4 +367,39 @@ void main() {
       );
     },
   );
+
+  test(
+      'Given a class with a field name longer than 63 characters, then an error is collected.',
+      () {
+    var models = [
+      ModelSourceBuilder().withYaml(
+        '''
+        class: Example
+        fields:
+          thisIsAVeryLongFieldNameThatIsLongerThan63CharactersAndThereforeInvalid: String
+        ''',
+      ).build()
+    ];
```

Comment:
Suggestion: Make a test at 63 chars to make sure it passes! And then change the failing tests to be exactly 64 chars, so that we validate the exact breaking point / edge. In the case someone fucks up the if statement in the future we have full coverage!

This applies for all the tests in this commit!